onlinejudge/_implementation/utils.py

```diff
@@ -144,7 +144,8 @@ def exec_command(command_str: str, *, stdin: IO[Any], timeout: Optional[float] =
         if gnu_time is not None:
             command = [gnu_time, '-f', '%M', '-o', fh.name, '--'] + command
         if os.name == 'nt':
-            command = command_str  # type: ignore
+            # HACK: for multi thread unstable
```

Comment:
```suggestion
            # HACK: without this encoding and decoding, something randomly fails with multithreading; see https://github.com/kmyk/online-judge-tools/issues/468
```

コメントを丁寧にしておきたいです。 issue にリンクを貼っておくのがよいかなと思います。